src/order-history/_style.scss

```diff
@@ -0,0 +1,13 @@
+.page__order-history {
```

Comment:
Is `page__` a convention we want to write up somewhere?  Is namespacing it with `order-history` enough?

Reply:
For now it is for sure. I think it would be good to be clear on what is a namespace vs just a name. You could imagine .order-history being used somewhere else in our system for a totally different purpose.

Not quite sure where to document this. I'll make a note for now.